lib/tasks/feed.rake

```diff
@@ -0,0 +1,27 @@
+require_relative '../../../app/workers/feeds/all_worker'
+require_relative '../../../app/workers/feeds/feed_worker'
+
+desc 'Exports a JSON feed of yesterday to s3'
+task :feed, [:feed_name] => :environment do |_task, args|
+  if args[:feed_name].blank?
+    puts "No feed name provided. Valid feed names are: all, #{Feeds::AllWorker.feed_names.join(', ')}"
+    next
+  end
+
+  feed_name = args[:feed_name].downcase
+  unless feed_name == 'all' || Feeds::AllWorker.feed_names.include?(feed_name)
+    puts "#{feed_name} is not a valid feed name. Valid feed names are: all, #{Feeds::AllWorker.feed_names.join(', ')}"
+    next
+  end
+
+  Time.zone = 'London'
+  date = ENV['REPORT_ON_DATE'] ? Date.parse(ENV['REPORT_ON_DATE']) : Time.zone.yesterday
+
+  if feed_name == 'all'
+    Feeds::AllWorker.feed_names.each { |fn| Feeds::FeedWorker.new.perform(fn, date) }
+  else
```

Comment:
to ensure the job runs as a Sidekiq job, use `JobName.perform_later(a,b,c...)` rather than `JobName.new.perform(a,b,c...)` (i.e. perform_later class method, not perform instance method)

Reply:
I'm a little undecided on this, the idea behind it was that it'd run the jobs while you wait, rather than having to check manually in sidekiq after to see if they ran successfully. I'm happy to change it to schedule the jobs instead though